### DISCUSSION TITLE
Blog 的延续 - 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#6] [Announcements]
# Blog 的延续 - 连波的闲谈杂鱼

最近花了一些时间将我的博客迁移到了 Astro 平台，我感觉整个过程挺美妙的，很喜欢 Astro。

http://godruoyi.com/posts/blog-3/

**Comment by godruoyi** [comment]
hello

**Reply by godruoyi** [reply]
hello 2

**Reply by godruoyi** [reply]
hello3

**Comment by Emt-lin** [comment]
> 我写博客的初衷是为了多年后能自省当年的自己，看看当时的心态。我比较厌烦流水账的记录，所以更多的时候我愿意写下此刻的感悟；

我也喜欢这样。

> 我不明白为什么很多人的年终总结要写一月份干了什么二月份干了什么，十年后的你在回首时真的在乎你某年某月干了什么吗？

可能更多是为了分享欲？因为网络上的人都不认识，可能更好的表达自己。

**Reply by godruoyi** [reply]
有分享欲并且善于通过文字表达出来的人已经很好了，至于文字内容即风格确实因人而异了，有可能即使是流水账，对于别人来说也是有价值的，👍

**Comment by linchen1987** [comment]
hello4

**Comment by ppd0705** [comment]
我觉得客观记录也是好的

**Reply by godruoyi** [reply]
确实，是我太主观认为了，每个人写东西都有自己想记录的事，动笔就是很好的了。

**Comment by syhily** [comment]
错别字：静态话的博客
应该改为：静态化的博客

**Reply by godruoyi** [reply]
呀，谢谢大佬，马上改。
